source/ota_mqtt.c

```diff
@@ -687,15 +689,20 @@ static uint32_t prvBuildStatusMessageFinish( char * pMsgBuffer,
                                              size_t msgBufferSize,
                                              OtaJobStatus_t status,
                                              int32_t reason,
-                                             int32_t subReason )
+                                             int32_t subReason,
+                                             uint32_t updaterVersion )
```

Comment:
Inconsistent with the above declaration ie `previousVersion`

Reply:
https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/232/commits/fcd5a6e66b056dbed4623bdf68f9bb90e061843c